redfish-core/lib/account_service.hpp

```diff
@@ -1680,91 +1726,126 @@ inline void triggerUnauthenticatedACFUpload(
         return;
     }
 
+    std::optional<nlohmann::json> acf;
     if (ibm)
     {
-        std::optional<nlohmann::json> acf;
         if (!redfish::json_util::readJson(*ibm, asyncResp->res, "ACF", acf))
         {
             BMCWEB_LOG_ERROR << "Illegal Property ";
             messages::propertyMissing(asyncResp->res, "ACF");
             return;
         }
+    }
```

Comment:
This doesn't look right, see: https://github.com/openbmc/bmcweb/blob/613dabea03cd2f5cb999be37aaf539280c63ea7a/redfish-core/lib/managers.hpp#L2231

See how it is 
```
        if (oem)   <- layer 1
        {
               std::optional<nlohmann::json> openbmc; <- now let's peel layer 2 
              if (!redfish::json_util::readJson(*oem, asyncResp->res, "OpenBmc",
                                              openbmc))
            {
                BMCWEB_LOG_ERROR
                    << "Illegal Property "
                    << oem->dump(2, ' ', true,
                                 nlohmann::json::error_handler_t::replace);
                return;
            }
            if (openbmc)
            {
                std::optional<nlohmann::json> fan;
                if (!redfish::json_util::readJson(*openbmc, asyncResp->res,
                                                  "Fan", fan))                                              
                                              
```

Let's follow that pattern